_pages/fsa/install/fsa-schematics.md

```diff
@@ -54,7 +54,7 @@ The following is a description of the various options you can use with the `ng a
 To have a clear picture on how some of these properties can be used, user can enter the following command that sets baseSite(s), currency(ies), and language(s):
 
 ```shell
-ng add @spartacus/fsa-schematics --baseSite=sample-financial-site --currency=usd,eur --language=en,de,fr
+ng add @spartacus/fsa-schematics --base-site=sample-financial-site --currency=usd,eur --language=en,de,fr
```

Comment:
Revert changes in this file. It's doc for an older version of Spartacus. 

Although it is not in one of the archived doc site version, the page says it's for spartacus 3.3